### DISCUSSION TITLE
Boss Health Bar Threshold Indicators

### DISCUSSION THREAD

**Discussion by NightsCreed** [runelite/runelite#16666] [Ideas]
Wondering if there's a possibility to add static threshold bars to boss or designated NPC health bars, similar to vanguards at COX.  This would be a visual indication of boss phase thresholds, set manually at specific HP or HP percentage intervals.  I'm new to PVM, but some of the instances that could be useful would be Alchemical Hydra, Hunllef, Baba, etc.

**Comment by Alexsuperfly** [comment]
The boss health indicators plugin on the hub probably does this

**Reply by NightsCreed** [reply]
Literally found it a minute after I posted, thanks!
